.github/workflows/test.yml

```
@@ -97,4 +99,34 @@ jobs:
         # Although "base" has a additional commit modifying the file matching  "testzone_1/*.txt" (testzone_1/added_by_other_pr.txt),
         # the file is already merged in the "base"
         run: '[[ "${{ steps.filter.outputs.test }}" = "false" && "${{ steps.filter.outputs.this_pr }}" = "true" ]]'
-    
\ No newline at end of file
+  test-4:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Committer Info
+        run: |
+          git config user.name 'Github Action' && \
+          git config user.email 'action@github.com'
+      # add a fake commit
+      - name: Add Commit
+        run: |
+          touch a.txt && \
+          touch b.txt && \
+          git add a.txt b.txt && \
+          git commit -m "add txt files"
+      - uses: ./
+        id: filter
+        with:
+          filters: |
+            test:
+              - *.txt
```

The test failed. 
It seems that the YAML parse recognizes this as an YAML alias, quoting the string should fix that.
```suggestion
              - '*.txt'
```

---

Thank you for investigating

As this isn't stated in the documentation you might want to add it since `*.<extension>` is a common pattern